src/ConsoleCommand.php

```diff
@@ -18,6 +19,7 @@ public function __construct(
         /** @var string[] */
         public readonly array $aliases = [],
         public readonly ?string $help = null,
+        public readonly ?CronDefinition $cron = null,
```

Comment:
I think `schedule` is a better word than `cron`, but I'm not entirely against `cron` either. wdyt?